src/main/java/com/playtika/homework1/task3/Romb.java

```diff
@@ -0,0 +1,24 @@
+package com.playtika.homework1.task3;
+
+public class Romb {
+    public static void main(String[] args) {
+        for (int i = 0; i < 5; i++) {
```

Comment:
you should type the size of rhombus side in the console

Reply:
fixed